app/code/Magento/SalesInventory/Observer/RefundOrderInventoryObserver.php

```diff
@@ -18,59 +22,23 @@
  */
 class RefundOrderInventoryObserver implements ObserverInterface
 {
-    /**
-     * @var StockConfigurationInterface
-     */
-    private $stockConfiguration;
-
-    /**
-     * @var StockManagementInterface
-     */
-    private $stockManagement;
-
-    /**
-     * @var \Magento\CatalogInventory\Model\Indexer\Stock\Processor
-     */
-    private $stockIndexerProcessor;
-
-    /**
-     * @var \Magento\Catalog\Model\Indexer\Product\Price\Processor
-     */
-    private $priceIndexer;
-
-    /**
-     * @var \Magento\SalesInventory\Model\Order\ReturnProcessor
-     */
-    private $returnProcessor;
-
-    /**
-     * @var \Magento\Sales\Api\OrderRepositoryInterface
-     */
-    private $orderRepository;
-
     /**
      * RefundOrderInventoryObserver constructor.
      * @param StockConfigurationInterface $stockConfiguration
      * @param StockManagementInterface $stockManagement
-     * @param \Magento\CatalogInventory\Model\Indexer\Stock\Processor $stockIndexerProcessor
-     * @param \Magento\Catalog\Model\Indexer\Product\Price\Processor $priceIndexer
+     * @param StockProcessor $stockIndexerProcessor
+     * @param PriceProcessor $priceIndexer
      * @param ReturnProcessor $returnProcessor
-     * @param \Magento\Sales\Api\OrderRepositoryInterface $orderRepository
+     * @param OrderRepositoryInterface $orderRepository
      */
     public function __construct(
-        StockConfigurationInterface $stockConfiguration,
-        StockManagementInterface $stockManagement,
-        \Magento\CatalogInventory\Model\Indexer\Stock\Processor $stockIndexerProcessor,
-        \Magento\Catalog\Model\Indexer\Product\Price\Processor $priceIndexer,
-        \Magento\SalesInventory\Model\Order\ReturnProcessor $returnProcessor,
-        \Magento\Sales\Api\OrderRepositoryInterface $orderRepository
+        private readonly StockConfigurationInterface $stockConfiguration,
+        private readonly StockManagementInterface $stockManagement,
+        private readonly StockProcessor $stockIndexerProcessor,
+        private readonly PriceProcessor $priceIndexer,
+        private readonly ReturnProcessor $returnProcessor,
+        private readonly OrderRepositoryInterface $orderRepository
```

Comment:
A trailing comma makes future additions easier to review.
```suggestion
        private readonly OrderRepositoryInterface $orderRepository,
```